engine/lib/output.php

```diff
@@ -142,6 +142,11 @@ function elgg_format_attributes(array $attrs = array()) {
 	$attributes = [];
 
 	foreach ($attrs as $attr => $val) {
+		if (!is_string($attr) || $attr[0] == '#') {
```

Comment:
I think I am going to change my position here. Let's filter out field vars until we have a compelling use case to pass them on